Woof/Woof/View/SitterFlow/SitterProfileView/SitterInfoCardView.swift

```diff
@@ -0,0 +1,107 @@
+import SwiftUI
+
+struct SitterInfoCardView: View {
+    /// The name of the pet sitter.
+    let name: String
+
+    /// The surname of the pet sitter.
+    let surname: String
+
+    /// The phone of the pet sitter.
+    let phone: String
+
+    /// The brief information about the pet sitter.
+    let bio: String
+
+    /// The price charged by the pet sitter per hour for dog walking.
```

Comment:
Nit: maybe in feature, as we've discussed, a pet sitter can go for a walk with other animals.
```suggestion
    /// The price charged by the pet sitter per hour for walking.
```

Reply:
Fixed.